rest-api/test/unit_test/api_test/awardee_api_test.py

```diff
@@ -70,7 +81,8 @@ def test_get_awardee_no_organizations(self):
   def test_get_awardee_with_organizations(self):
     self._setup_data()
     result = self.send_get('Awardee/PITT')
-    self.assertEquals(self._make_expected_pitt_awardee_resource(), result)
+    # we are now filtering out 'INACTIVE' sites by default.
+    self.assertNotEqual(self._make_expected_pitt_awardee_resource(), result)
```

Comment:
We shouldn't be asserting that these aren't equal... we should be asserting what we actually expect to get returned.

I would make _make_expected_pitt_awardee_resource() take a param indicating whether to include inactive sites.